api/config.py

```diff
@@ -266,16 +266,19 @@ class Configuration(CoreConfiguration):
             "description": _(
                 "An email address a patron can use if they need help, e.g. 'simplyehelp@yourlibrary.org'."
             ),
-            "required": True,
+            # "required": True,
```

Comment:
Was this supposed to be removed?

Reply:
Yes, patron support email is no longer a required attribute. It's now a special case where either support email or support website must be used

Reply:
Sorry, what I meant was: Was this supposed to be _removed_ instead of just commented out?

Reply:
Yea, I generally flip flop on this while coding. Removed